README.md

```diff
@@ -155,4 +166,3 @@
 
 * [Landing Page Optimization](https://www.google.com/url?q=https://books.google.com.ph/books/about/Landing_Page_Optimization.html?id%3DpnL4JRg7z1cC%26source%3Dkp_cover%26redir_esc%3Dy&sa=D&ust=1507602583004000&usg=AFQjCNFB-z6ANZkwPnBDvUyuGbFuF5pp2Q): A fully updated guide to making your landing pages profitable.
```

Comment:
can you add the direct link instead of a google redirect.

Reply:
will be changed in updated pull request.